toonz/sources/toonz/xshrowviewer.cpp

```diff
@@ -149,16 +158,14 @@ void RowArea::drawRows(QPainter &p, int r0, int r1) {
         str = QString("%1\"").arg(QString::number(koma).rightJustified(2, '0'));
       }
 
-      p.drawText(QRect(width() / 2 - 15, y + 1, width() / 2 + 7, RowHeight - 2),
-                 Qt::AlignRight | Qt::AlignBottom, str);
+      p.drawText(labelRect, align, str);
```

Comment:
Fixed. Modified settings in orientation.cpp.  In timeline mode, frame label is centered in column and in xsheet mode, margins restored